packages/namekit-react/tailwind.config.ts

```diff
@@ -3,6 +3,14 @@ import type { Config } from "tailwindcss";
 const config: Config = {
   content: ["src/**/*.{ts,tsx,js,jsx,mdx}"],
   prefix: "nk-",
+  theme: {
+    extend: {
+      colors: {
```

Comment:
Is this the best practice for us to manage colors?

Reply:
@lightwalker-eth indeed. This colour isn't a Tailwind default, so we need to add it here so we can use it. [Name That Color](https://chir.ag/projects/name-that-color) is a great method for naming colors so I used that here.

Reply:
@notrab ok awesome, thank you :+1: